src/modules/chat_font_settings/index.js

```diff
@@ -12,9 +12,17 @@ Leave the field blank to use the default.`;
 
 const STYLE_ID = 'bttv-font-size';
 
+const encodeFontFamily(fontFamily) => {
+    if (['serif', 'sans-serif', 'monospace', 'cursive', 'fantasy', 'system-ui'].indexOf(fontFamily)) {
```

Comment:
indexOf returns `-1` when it doesn't find anything .. this evaluates to true when it doesn't find anything

Reply:
there should also be a constant made of all generic names